setup.py

```diff
@@ -34,7 +34,17 @@
             [
                 os.path.join(classic_folder, a_file)
                 for a_file in os.listdir(classic_folder)
-                if os.path.isfile(os.path.join(classic_folder, a_file)) and not a_file.endswith(".py")
+                if os.path.isfile(os.path.join(classic_folder, a_file))
+                and not a_file.endswith(".py")
+            ]
+        ),
+        (
+            "share/jupyter/nbextensions/mamba_gator/static",
+            [
+                os.path.join(classic_folder, "static", a_file)
+                for a_file in os.listdir(os.path.join(classic_folder, "static"))
+                if os.path.isfile(os.path.join(classic_folder, "static", a_file))
+                and not a_file.endswith(".py")
```

Comment:
Wondering if this could be handled with a glob pattern, that would exclude .py files. Globs should be supported by `jupyter-packaging` if we end up switching to it in the near future (for example when [updating to lab 3.0](https://github.com/mamba-org/gator/pull/111)).

Reply:
It might also make it easier to handle nested folders in `static` (if any)